lib/log_ratelimit.c

```diff
@@ -35,6 +35,14 @@ struct log_ratelimit_state {
 
 static struct log_ratelimit_state log_ratelimit_states[RTE_MAX_LCORE];
 
+static bool enabled = false;
```

Comment:
Global variables are zeroed automatically during the load of the process, so drop the initialization.

At least in the past, there used to be a penalty when a global variable is explicitly initialized with zeros. Moderns compilers may be able to overcome the problem automatically, but we don't need to rely on that.